etc/adminhtml/system.xml

```diff
@@ -334,8 +334,9 @@
                         <comment><![CDATA[<b>Advanced.</b> Overrides the <code>customer_cancel_url</code> value of created payments]]></comment>
                         <config_path>payment/alma_monthly_payments/customer_cancel_url</config_path>
                     </field>
-                    <field id="share_checkout_enable" translate="label" type="select" sortOrder="17" showInDefault="1">
-                        <label>Enabled Share of Checkout</label>
+                    <field id="share_checkout_enable" translate="label comment" type="select" sortOrder="17" showInDefault="1">
```

Comment:
why did you add `comment` after `label` in `translate` attribute ?

Reply:
To also translate the comment.é